docs/tutorial/03-resources.mdx

```diff
@@ -0,0 +1,488 @@
+        return "Hello, World!"
+    }
+}
+```
+
+### Resources
+
+The key difference between a resource and a struct or class is the access scope for resources:
+
+- Each instance of a resource can only exist in exactly one location and cannot be copied.
+  Here, location refers to account storage, a temporary variable in a function, a storage field in a contract, etc.
+- Resources must be explicitly moved from one location to another when accessed.
+- Resources also cannot go out of scope at the end of function execution.
+  They must be explicitly stored somewhere or destroyed.
+
+These characteristics make it impossible to accidentally lose a resource from a coding mistake.
+
+```cadence
+init() {
+// ...
+```
+
+All composite types like contracts, resources, and structs can have an optional `init()` function that only runs when the object is initially created.
+Cadence requires that all fields must be explicitly initialized, so if the object has any fields, this function has to be used to initialize them.
+
+Contracts also have read and write access to the storage of the account that they are deployed to by using the built-in [`self.account`](/cadence/language/contracts) object.
+This is an [`AuthAccount` object](/cadence/language/accounts/#authaccount)
+that gives them access to many different functions to interact with the private storage of the account.
+
+This contract's `init` function is simple, it logs the phrase `"Hello Asset"` to the console.
+
+
+**A resource can only be created in the scope that it is defined in.**
+
+This prevents anyone from being able to create arbitrary amounts of resource objects that others have defined.
+
+### The Move Operator (`<-`)
+
+In this example, we declared a function that can create `HelloAsset` resources:
+```cadence
+pub fun createHelloAsset(): @HelloAsset {
+    return <-create HelloAsset()
+}
+```
+The `@` symbol specifies that it is a resource, of the type `HelloAsset` which we defined in the contract.
+This function uses the move operator to create a resource of type `HelloAsset` and return it
+To create a new resource object, we use the `create` keyword
+
+Here we use the `<-` symbol. [This is the move operator](/cadence/language/resources/#themoveoperator).
+The move operator `<-` replaces the assignment operator `=` in assignments that involve resources.
+To make the assignment of resources explicit, the move operator `<-` must be used when:
+
+- the resource is the initial value of a constant or variable,
+- the resource is moved to a different variable in an assignment,
+- the resource is moved to a function as an argument
+- the resource is returned from a function.
+
+When a resource is moved, the old location is invalidated, and the object moves into the context of the new location.
+
+So if I have a resource in `first_resource`, like so:
+
+```cadence
+// Note the `@` symbol to specify that it is a resource
+let first_resource: @AnyResource = create AnyResource()
```

Comment:
Are you sure that the above statement is correct? shouldn't it move `<-` operator used instead of `=`

Reply:
yes, good catch